.github/workflows/linux-build.yml

```
@@ -44,15 +40,123 @@ on:
+          rm -rf /opt/ghc
+          rm -rf /opt/hostedtoolcache/CodeQL
+          rm -rf "/usr/local/share/boost"
+          rm -rf "$AGENT_TOOLSDIRECTORY"
+          docker image prune --all --force
+          docker builder prune -a
+          df -h
+      - name: Get Ccache Stash
+        uses: assignUser/stash/restore@v1
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-ubuntu-debug-default-${{ inputs.use-clang && 'clang' || 'gcc' }}
```

actionlint (1.7.7)

property "use-clang" is not defined in object type {}

(expression)